infra/main.test.bicep

```diff
@@ -0,0 +1,17 @@
+// This file is for doing static analysis and contains sensible defaults
+// for PSRule to minimise false-positives and provide the best results.
+
+// This file is not intended to be used as a runtime configuration file.
+
+targetScope = 'subscription'
+
+param environmentName string = 'testing'
+param location string = 'swedencentral'
```

Comment:
Should we use Sweden central here or East US 2 as well?

Reply:
As long as it's one of the allowed regions,it doesn't matter